lib/mayaUsd/nodes/proxyShapeBase.cpp

```diff
@@ -605,41 +605,31 @@ MStatus MayaUsdProxyShapeBase::computeInStageDataCached(MDataBlock& dataBlock)
                 loadSet = UsdStage::InitialLoadSet::LoadNone;
             }
 
-            {
-                // When opening or creating stages we must have an active UsdStageCache.
-                // The stage cache is the only one who holds a strong reference to the
-                // UsdStage. See https://github.com/Autodesk/maya-usd/issues/528 for
-                // more information.
-                UsdStageCacheContext ctx(
```

Comment:
A stage cache is needed for the re-use of stages across proxy shapes. Issue #528 was logged when we worked on in-memory stage workflows, and here leveraging stage cache doesn't make much sense, since we are not going to re-use that stage in another proxy.

Your code should look like what was there before https://github.com/Autodesk/maya-usd/pull/519/commits/4af6c19ba9a4e4984730ee096792d48d196b7b24, i.e. UsdStageCacheContext is used when we have a layer from file and need to create a stage for it.

Having said all of that, I know we have an internal ticket (MAYA-109039) that will require changes in the same place. How about we close this PR and make the change as part of MAYA-109039?